src/WebpubViewer.ts

```diff
@@ -1,18 +1,22 @@
+import Store from "./Store";
 import Cacher from "./Cacher";
 import Navigator from "./Navigator";
 import Annotator from "./Annotator";
 
 /** Viewer application for web publications. */
 export default class WebpubViewer {
+    private store: Store;
     private cacher: Cacher;
     private navigator: Navigator;
     private annotator: Annotator | null;
 
     /** Create a WebpubViewer. */
+    /** @param store A Store to save application settings or data. */
     /** @param cacher A Cacher to handle storing webpub and app resources for offline use. */
     /** @param navigator A Navigator to handle moving between spine items and resources of the publication. */
     /** @param annotator Optional Annotator that handles bookmarking, last reading position, highlights. */
-    public constructor(cacher: Cacher, navigator: Navigator, annotator: Annotator | null = null) {
+    public constructor(store: Store, cacher: Cacher, navigator: Navigator, annotator: Annotator | null = null) {
```

Comment:
Yes, for now, but I will probably have to add another optional service soon and then I won't be able to do that.